src/EventGrid/EventGrid/help/Get-AzEventGridSystemTopic.md

```diff
@@ -0,0 +1,158 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.EventGrid.dll-Help.xml
+Module Name: Az.EventGrid
+online version:
+schema: 2.0.0
+---
+
+# Get-AzEventGridSystemTopic
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
+
+## SYNTAX
+
+### ResourceGroupNameParameterSet (Default)
+```
+Get-AzEventGridSystemTopic [[-ResourceGroupName] <String>] [-ODataQuery <String>] [-Top <Int32>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### SystemTopicNameParameterSet
+```
+Get-AzEventGridSystemTopic [[-ResourceGroupName] <String>] [[-Name] <String>] [-ODataQuery <String>]
+ [-Top <Int32>] [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### NextLinkParameterSet
+```
+Get-AzEventGridSystemTopic [-NextLink <String>] [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+{{ Fill in the Description }}
+
+## EXAMPLES
+
+### Example 1
+```powershell
```

Comment:
A good example should be a good start for customer

Reply:
fixed